### ISSUE TITLE
ng-cli appears to install but does not run on Windows 10

### ISSUE THREAD

**Issue by MrRhino** [angular/angular#17071]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
When trying to run the command "ng new my-app" i get this error message:
'ng' is not recognized as an internal or external command, operable program or batch file.

This is also reported by lots of others apparently having the same issue.

**Expected behavior**
The command "ng new my-app" should execute without any errors.

**Minimal reproduction of the problem with instructions**
On a Windows 10 machine:
Follow the cli-quickstart https://angular.io/docs/ts/latest/cli-quickstart.html

**What is the motivation / use case for changing the behavior?**
Trying to get started with the latest Angular 2 version

**Please tell us about your environment:**
CMD on Windows 10

* **Angular version:** 2.0.X
Latest Angular 2

* **Browser:**
Not relevant
 
* **Language:**
Not relevant

* **Node (for AoT issues):** `node --version` =   v7.10.0

**Comment by jotatoledo** [comment]
The issue tracker for @angular/cli is [here](https://github.com/angular/angular-cli). Consider moving the issue to the right repo

**Comment by jotatoledo** [comment]
How did u installed the cli? `npm install -g @angular/cli` or `npm install @angular/cli`?

**Comment by MrRhino** [comment]
I installed with the -g option for global: "npm install -g @angular/cli"

**Comment by ocombe** [comment]
you can have this problem if you install it with both yarn & npm, uninstall in both and reinstall only with npm (if that was the case)

**Comment by MrRhino** [comment]
Hi Olivier,

I'm just using npm on Windows 10.
Today i uninstalled node restarted and 7.10.0 and restarted it.
But i still get *'ng' is not recognized as an internal or external command,
operable program or batch file.*

Best Regards

Ulrik Sorber

**Comment by MrRhino** [comment]
I have the same issue.
You can not install ng from the command line with a non administrator account
And there is no sudo. Only runas which runs as the administrator with the administrator profile.
The UAC does not kick in and ask you to provide admin permissions as when you run an installer.
So the ng binary will be located in the Administrator npm folder after installing using the runas feature.

Take a look at the folder "C:\Users\Administrator\AppData\Roaming\npm" and see if you can spot ng.
If you can then add "C:\Users\Administrator\AppData\Roaming\npm" to your path.

There is something called sudo for windows ( https://sourceforge.net/projects/sudowin/ )
It runs with elevated permissions, while preserving your profiling settings.
I haven't tried it. Use at own risk.

**Comment by MrRhino** [comment]
I just responded to the same issue under angular/angular-cli [(https://github.com/angular/angular-cli/issues/6501)]
with the following comment.

You can not install ng from the command line with a non administrator account
And there is no sudo. Only runas which runs as the administrator with the administrator profile.
The UAC does not kick in and ask you to provide admin permissions as when you run an installer.
So the ng binary will be located in the Administrator npm folder after installing using the runas feature.

Take a look at the folder "C:\Users\Administrator\AppData\Roaming\npm" and see if you can spot ng.
If you can then add "C:\Users\Administrator\AppData\Roaming\npm" to your path.

There is something called sudo for windows ( https://sourceforge.net/projects/sudowin/ )
It runs with elevated permissions, while preserving your profiling settings.
I haven't tried it. Use at own risk.

**Comment by saharpure** [comment]
any resolution on this issue? getting same problem on Windows 10, after running `npm install @angular/cli` -g getting ng not recognized.

**Comment by skm2017** [comment]
I faced the same issue after installing angular cli with `npm install -g @angular/cli` command on windows 10. 
Then I applied the solution offered by [MrRhino](https://github.com/angular/angular/issues/17071#issuecomment-304740645) and it worked.

> Take a look at the folder "C:\Users\your-user-folder\AppData\Roaming\npm" and see if you can spot ng. If you can, then add "C:\Users\your-user-folder\AppData\Roaming\npm" to your path.

If you have any cmd open then close them and then reopen and try with `ng` command.

Hope this helps someone.
